tasks/src/object-detection/data.ts

```diff
@@ -7,7 +7,7 @@ const taskData: TaskData = {
 	datasets: [
 		{
 			// TODO write proper description
-			description: "Benchmark dataset used for object detection",
+			description: "The COCO dataset is widely used as a benchmark for models employed for image segmentation, object detection, and captioning",
```

Comment:
Same as merve's comment above, this description is too long. (see suggestion in the other comment)